auto-notify.plugin.zsh

```diff
@@ -51,24 +61,45 @@ function _auto_notify_message() {
     body="$(_auto_notify_format "$text" "$command" "$elapsed" "$exit_code")"
 
     if [[ "$platform" == "Linux" ]]; then
+        # Set default notification properties
         local urgency="normal"
-        local transient="--hint=int:transient:1"
-        local icon=${AUTO_NOTIFY_ICON_SUCCESS:-""}
-        # Exit code 130 is returned when a process is terminated with SIGINT.
-        # Since the user is already interacting with the program, there is no
-        # need to make the notification persistent.
-        if [[ "$exit_code" != "0" ]] && [[ "$exit_code" != "130" ]]; then
+        local transient="--hint=int:transient:$AUTO_NOTIFY_ENABLE_TRANSIENT"
+        local icon="${AUTO_NOTIFY_ICON_SUCCESS:-""}"
+
+        # Handle specific exit codes
+        if [[ "$exit_code" -eq 130 ]]; then
+            # Exit code 130 indicates termination by SIGINT (Ctrl+C).
+            # If AUTO_NOTIFY_CANCEL_ON_SIGINT is enabled, suppress the notification.
+            if [[ "${AUTO_NOTIFY_CANCEL_ON_SIGINT}" -eq 1 ]]; then
+                return
+            fi
```

Comment:
@MichaelAquilina Why is this guarded by the `if [[ "$platform" == "Linux" ]]` condition? There is no reason to treat macOS (`"$platform" == "Darwin"`) differently